AntonEmery/main.js

```diff
@@ -0,0 +1,48 @@
+// Your card and deque factory code should be loaded into the browser before this file
```

Comment:
Looks good up through 2b! Let me know if you have any questions about the later sections; always happy to have a look.